jwql/website/apps/jwql/views.py

```diff
@@ -1151,7 +1151,10 @@ def view_image(request, inst, file_root, rewrite=False):
             suff_arr = np.array(image_info['suffixes'])
             files_arr = np.array(image_info['all_files'])
             splits = np.array([ele.split('_')[-1] for ele in image_info['suffixes']])
-            idxs = np.where(splits == poss_suffix)[0]
+            if splits.size > 0:
```

Comment:
Was this intended to be included in this PR?

Reply:
Yes, this fixes an issue that throws an error in the unit tests later numpy versions.